samples/ipc_telemetry/README.md

```diff
@@ -0,0 +1,22 @@
+# Sample ipc_telemetry
+
+## Simple producer Consumer sample using IPC telemetry.
+
+This sample is composed by 3 different actors:
+
+1. **Consumers**: The actor that will subscribe for telemetry in the sensors interface. Those are
+                    the codes in the `consumers` directory.
+2. **Producers**: The actor that will expose the sensor interface with telemetry notification.
+                    Those are the codes in the `producers` directory.
+3. **OS**: This is the glue between `producers` and `consumers`, it create an instance of the IPC,
```

Comment:
nit: This is the glue between `producers` and `consumers`. It creates an instance of the IPC, and
                    installs the producer that provides telemetry for multiple sensors and
                    the consumer that subscribes to receive these telemetries.